data/classes/fx.json

```diff
@@ -1,17 +1,26 @@
 {
   "name": "fx",
+  "javadoc": [
+    "A position of a block in 3 dimensions."
+  ],
   "fields": [
     {
       "name": "a",
-      "descriptor": "Lcom/mojang/serialization/Codec;"
+      "descriptor": "Lcom/mojang/serialization/Codec;",
+      "javadoc": [
+        "The codec for a {@code BlockPos}."
+      ]
```

Comment:
This is more of my opinion, but if a codec is javadoced, the format in which the data is preserved is useful to know. For example, the block pos serializes to/deserializes from an integer array of three elements.

Reply:
*Well akcuthally*, all you can specify is that it uses the `Codec.INT_STREAM`. Which invokes `DynamicOps#getIntStream`. Which itself is implementation specific. Yes, both `JsonOps` and `NBTDynamicOps` map this down to an int array when serialized to a data format. But codec's are data-format-agonstic. So this codec itself does not say it serializes to a int array *nessecarilly.*

Reply:
If I wanted to correct my terminology, it would be more appropriate to say an int list of three elements. Either way, without the specific information on the expected format, this too would also violate simple/expected knowledge.

Reply:
Except it doesn't require it being a list. You could, in theory, implement a dynamic ops that serialized streams as:
```json
{
    "stream this baby": true,
    "first": 0,
    "second": 2,
    "third": 2
}
```
You'd be quite stupid if you did, but you could. *I'll shutup now*

Reply:
For now, I've removed the javadoc for the codec field, since my change adds no real substantive information.